### ISSUE TITLE
"show log" - log lines truncated to 512 characters

### ISSUE THREAD

**Issue by jpstotz** [paralect/robomongo#1062]
When using the "show log" command on a remote Mongo server the resulting log lines are truncated to a maximum length of 512 characters. 

The problem is that most log entries (e.g. of slow queries) are longer than 512 characters.
Therefore from my perspective the "show log" command is of very limited use.

Is there a way to increase the line length limit?

**Comment by anorsich** [comment]
Hi @jpstotz! 

Thank for reporting this issue. We were able to replicate it. Robomongo does not truncate error messages, but there is an issue with a horizontal scroll in a text mode. For some reason, it won't go more than 512 characters. But if you set a cursor to the end of a long line and do a couple of right arrow clicks you will see an entire message. 
As another, temporary, workaround you can just copy entire log and paste to any other text editor. 

Steps to replicate:
1. execute `show logs` command
2. find a line, which is more than 512 characters, try to scroll till the end of line. 
Result:
You can't scroll horizontally till the end of the line.

**Comment by jpstotz** [comment]
Hi anorsich, thanks for your answer.

On which OS and Robomongo version did you test this? Because on Windows 7 using Robomongo 0.9.0-rc6 pressing the "end" key brings me not to the end of the line but to column 512 (on longer lines). If I now press the right cursor key the view switches to the start of the next line.

Also if I select a complete line or everything, copy it to the clipboard and paste it into an editor all lines are truncated to 512 characters. 
Therefore from my point of view it is more than just a scroll problem.

Edit: I am able to reproduce the "scroll problem" you observed, but this happens already on column 250 on my system. Hence we have to independent problems...

**Comment by anorsich** [comment]
@jpstotz I am using OSX, El Capitan. Looks like this is another, windows specific issue, where lines get truncated. @juliashibalko Can we try to reproduce it on Windows?

**Comment by juliashibalko** [comment]
@anorsich Yes, I'm able to reproduce this problem on 0.9-RC7 under Windows 10. It's truncated the line on 512 column. 
Thanks @jpstotz for reporting the issue! Will work to fix this problem.
